### PR TITLE
update Tsinghua University part

### PR DESCRIPTION
**Pull request by sholsyjy**

### CODE REVIEW

File: content/zh-cn/docs/campus/quality.md [hunk 1]
```diff
@@ -532,7 +566,6 @@ _感谢<u>YJ Wang</u>的投稿_
 
 军训分男女寝，不剪头，尺码自订，2020 年与 2021 年军训地点在舒适的雁栖湖校区
 体育课不分男女，但内容可能会根据性别有微量区分
-editor所上的足球课，只要传球动作标准，分数即可90+
```
Comment (saeziae) [inline content/zh-cn/docs/campus/quality.md:535]: why you deleted this

File: content/zh-cn/docs/campus/quality.md [hunk 2]
```diff
@@ -362,6 +362,40 @@ MtF 分部情况：
...
+
+### 宿舍情况与基础设施
+
+清华因为有不同时间修建的宿舍，所以会出现比较复杂的情况。首先是老楼，宿舍是三人间，有本科生也有研究生。然后是紫荆公寓（本科），四人间，每两间之间有个中厅，男生宿舍厕所大概每8间一个的公共厕所有小便池和隔间，浴室也是大概每8间一个的大概有3个淋浴头的公共浴室。然后是南区宿舍，也是4人一间，没有中厅，是最新的宿舍，有独立浴室，相对私密性会好一些。然后是不同的研究生公寓，w楼（紫荆博士公寓）是两人一间，独立卫生间，两间屋共用一个阳台；其他研究生宿舍不清楚。
+
+### 医疗资源状况
+
+北医三院和北医六院是清华学生非常熟悉的地方了，也是trans们在北京基本上大概率会去的地方。去回龙观医院也很方便。
+
+重点讲一下校医院的情况——清华的学生是公费医疗，基本上在校医院花2块钱挂号，剩下的大概率都不用花钱。一个非常棒的是，内科的医生是对“异性症”三个字有概念的，基本上你说明自己的情况，医生也不会多问，在系统里选上“异性症”然后就给你开检查单了。然后是否能在校医院开出hrt相关的药品是未知的。
```
Comment (saeziae) [inline content/zh-cn/docs/campus/quality.md:387]: typo 易性症

**Review by saeziae — CHANGES_REQUESTED** [review]

File: content/zh-cn/docs/campus/quality.md (hunk 1 above)
Comment (sholsyjy) [reply content/zh-cn/docs/campus/quality.md:535]: mistake

**Comment by sholsyjy** [comment]
all fixed

**Pull request merged**